src/test/ui/const-generics/conservative_is_privately_uninhabited_uses_correct_param_env-1.rs

```diff
@@ -0,0 +1,27 @@
+// run-pass
```

Comment:
I think this should be check-pass.

Reply:
I don't think so. If i remember correctly this wasn't ICE'ing on `cargo check` only `cargo build`. Somewhat assuming `//run-pass` and `//check-pass` have similar meanings here

Reply:
I see. What about build-pass ? It doesn't run the bin after built like run-pass.